server/src/test/java/io/crate/types/CharacterTypeTest.java

```diff
@@ -38,22 +38,22 @@ public DataType<String> getType() {
     private static final SessionSettings SESSION_SETTINGS = CoordinatorTxnCtx.systemTransactionContext().sessionSettings();
 
     @Test
-    public void test_value_for_insert_adds_blank_padding() {
-        assertThat(CharacterType.of(5).valueForInsert("a")).isEqualTo("a    ");
+    public void test_default_length() {
+        assertThat(CharacterType.INSTANCE.lengthLimit()).isEqualTo(1);
     }
 
     @Test
-    public void test_implicit_cast_adds_blank_padding() {
-        assertThat(CharacterType.of(5).implicitCast("a")).isEqualTo("a    ");
+    public void test_value_for_insert_adds_blank_padding() {
+        assertThat(CharacterType.of(5).valueForInsert("a")).isEqualTo("a    ");
     }
 
     @Test
-    public void test_default_length() {
-        assertThat(CharacterType.INSTANCE.lengthLimit()).isEqualTo(1);
+    public void test_cast_adds_blank_padding() {
+        assertThat(CharacterType.of(5).explicitCast("a", SESSION_SETTINGS)).isEqualTo("a    ");
     }
 
     @Test
-    public void test_explicit_cast_truncates_overflow_chars() {
+    public void test_cast_truncates_overflow_chars() {
```

Comment:
Looks like there isn't any test for the implicit cast (which is the main change ;)?

Reply:
right, thought that since it calls explicit I should remove "noise", but It makes sense to add the tests for both and be future proof.